index.html

```diff
@@ -106,12 +106,6 @@ <h4>Key fingerprint=2BBE 2AA1 A641 F614 7B58 450B E352 7B60 DAAC A1D8</h4>
     <h2>MJ (Marcos JSP) @MarcosJSP</h2>
     <a href="keys/mj.asc">mj.asc</a>
     <a href="https://keys.openpgp.org/vks/v1/by-fingerprint/DBDD73C073C722E2A2B3CAD3F6D73AE37EBE96D2">0xF6D73AE37EBE96D2</a>
-    <h4>Key fingerprint=DBDD 73C0 73C7 22E2 A2B3 CAD3 F6D7 3AE3 7EBE 96D2</h4>
-    <hr />
-
-    <h2>Todd @tvangundy</h2>
-    <a href="keys/tvangundy.asc">tvangundy.asc</a>
-    <a href="https://keys.openpgp.org/vks/v1/by-fingerprint/C07B19078BB6DA872C8A0015B173BE29FD0DF36D">0xb173be29fd0df36d</a>
```

Comment:
Smth is wrong @rbhrzlabs. Key and fingerprint you are removing do not match :thinking: